Fix warning in tests/inlinert_float.c

I get this warning compiling

```
clang-11 -O3 -c -emit-llvm -o ./inlinert_unlinked_float.bc inlinert_float.c
inlinert_float.c:80:54: warning: integer literal is too large to be represented in a signed integer type, interpreting as unsigned [-Wimplicitly-unsigned-literal]
    testFloatToIntNoOverflow(-9223372036854775808.0, 9223372036854775808);
```

--- a/runtime/tests/inlinert_float.c
+++ b/runtime/tests/inlinert_float.c
@@ -77,7 +77,7 @@ void testFloatsToIntConvert() {
     testFloatToIntNoOverflow(-1.5, -2);
     testFloatToIntNoOverflow(-2.5, -2);
     testFloatToIntNoOverflow(9223372036854774784.0, 9223372036854774784);
-    testFloatToIntNoOverflow(-9223372036854775808.0, 9223372036854775808);
+    testFloatToIntNoOverflow(-9223372036854775808.0, -9223372036854775807 - 1);
 
     testFloatToIntOverflow(9223372036854775808.0);
     testFloatToIntOverflow(-9223372036854777856.0);

